flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+	"only-arches": [
+		"x86_64"
```

Comment:
disappointing.

Reply:
is this a joke?

Reply:
no. it's not. `aarch64` is nice to have.

Reply:
i have plans to add arm support in the future, however, that's going to require more collaboration with the upstream dev's as I don't believe they have official zip downloads that support  Linux arm devices yet (I'm hopeful though because they have it for Android and Apple Silicon)

Reply:
> Would it be preferable to remove this file and see if arm builds succeed on flathub infrastructure?

sure, that allows to test the build on both arches.

Reply:
> Would it be preferable to remove this file and see if arm builds succeed on flathub infrastructure?

sure, that allows to test the build on both arches.

Reply:
this has been done